libs/application/template-api-modules/src/lib/modules/templates/old-age-pension/old-age-pension-utils.ts

```diff
@@ -0,0 +1,110 @@
+import { Application, YES } from '@island.is/application/types'
+import {
+  ApplicationType,
+  getApplicationAnswers,
+} from '@island.is/application/templates/old-age-pension'
+import { parse } from 'date-fns'
```

Comment:
Breyta import til að fjarlægja eslint villu
```suggestion
import parse from 'date-fns/parse'
```